src/test/java/com/webank/webase/front/util/TransactionEncoderTest.java

```diff
@@ -0,0 +1,96 @@
+package com.webank.webase.front.util;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import java.math.BigInteger;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.List;
+
+import org.fisco.bcos.web3j.crypto.*;
+import org.fisco.bcos.web3j.crypto.gm.GenCredential;
+import org.fisco.bcos.web3j.rlp.RlpType;
+import org.fisco.bcos.web3j.utils.Numeric;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class TransactionEncoderTest {
+
+    static BigInteger gasPrice = BigInteger.ONE;
+    static BigInteger gasLimit = BigInteger.TEN;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/WeBankFinTech/WeBASE-Front/pullRequest?prid=4823531)